### PR TITLE
Document '-ios predicate string' locator strategy

### PR DESCRIPTION
**Pull request by dpgraham**
## Proposed changes

Updated documentation to show usage of -ios predicate string locator strategy

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://docs.google.com/forms/d/1lOfXRw_0VCk7gYzjj4WLetGu7yelDVo5LWh0z7pGftE/viewform)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### Reviewers: @imurchie, @jlipps, ...

Includes a Java example showing how to use predicate string in appium; documents the iOS version support

### CODE REVIEW

**Comment by triager** [comment]
Can one of the admins verify this patch?

**Comment by triager** [comment]
Can one of the admins verify this patch?

File: docs/en/writing-running-appium/ios_predicate.md [hunk 1]
```diff
@@ -13,11 +13,18 @@ It is worth looking at *'-ios uiautomation'* search strategy with **Predicates**
...
 // java
-appiumDriver.findElementsByIosUIAutomation("collectionViews()[0].cells().withPredicate(\"ANY staticTexts.isVisible == TRUE\")")
+appiumDriver.findElementsByIosUIAutomation("collectionViews()[0].cells().withPredicate(\"ANY staticTexts.isVisible == TRUE\")");
+```
+
+'-ios predicate string' example:
+
+```java
+// java
+appiumDriver.findElementsByIosNsPredicate("ANY staticTexts.isVisible == TRUE");
```
Comment (imurchie) [inline docs/en/writing-running-appium/ios_predicate.md]: This doesn't have to be `isWdVisible` or something?

Comment (dpgraham) [inline docs/en/writing-running-appium/ios_predicate.md]: I just cut and paste it. I probably should try running it.

Comment (imurchie) [inline docs/en/writing-running-appium/ios_predicate.md]: I'm just going by the tests you made in `appium-xcuitest-driver`.

Comment (dpgraham) [inline docs/en/writing-running-appium/ios_predicate.md]: I updated it to the test example I used.

**Comment by imurchie** [comment]
Cool. Can you squash this into a single commit? Then it's good to go!

**Pull request merged**